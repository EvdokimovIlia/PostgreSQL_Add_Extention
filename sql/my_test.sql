CREATE EXTENSION my_first_extension;

SELECT generate_check_digit('327');
SELECT generate_check_digit('816');
SELECT generate_check_digit('-16');
SELECT generate_check_digit('0');
SELECT generate_check_digit('Hi');

SELECT validate_check_digit('3283');
SELECT validate_check_digit('123');
SELECT validate_check_digit('0');
