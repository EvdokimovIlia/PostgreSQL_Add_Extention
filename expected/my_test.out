CREATE EXTENSION my_first_extension;
SELECT generate_check_digit('327');
 generate_check_digit 
----------------------
                 3283
(1 row)

SELECT generate_check_digit('816');
 generate_check_digit 
----------------------
                 8171
(1 row)

SELECT generate_check_digit('-16');
 generate_check_digit 
----------------------
                   -1
(1 row)

SELECT generate_check_digit('0');
 generate_check_digit 
----------------------
                    0
(1 row)

SELECT generate_check_digit('Hi');
 generate_check_digit 
----------------------
                    0
(1 row)

SELECT validate_check_digit('3283');
 validate_check_digit 
----------------------
 t
(1 row)

SELECT validate_check_digit('123');
 validate_check_digit 
----------------------
 f
(1 row)

SELECT validate_check_digit('0');
 validate_check_digit 
----------------------
 f
(1 row)

